### DISCUSSION TITLE
Pre release

### DISCUSSION THREAD

**Discussion by ronag** [nodejs/undici#723] [General]
Should we do a pre-release?

4.0.0-alpha? I’d like to start using it in our test setups.

**Comment by mcollina** [comment]
yes I think it's worthwhile.

**Comment by dnlup** [comment]
I agree too

**Comment by ronag** [comment]
How do I make a pre-release using npm?

**Reply by szmarczak** [reply]
https://github.com/sindresorhus/got/blob/72390c2dfb79f7ee0012d6ba264faff638248e47/package.json#L3

**Comment by ronag** [comment]
undici@4.0.0-alpha.0 published

**Comment by mcollina** [comment]
We should not tag those prereleases as “latest” but as “next”.

**Reply by ronag** [reply]
I’m not sure I know what that means.

**Reply by ronag** [reply]
Ah, dist-tag. That's new for me. I'll keep it in mind. @mcollina if you publish 3.x with your changes it should become latest again. I'll make sure to use dist-tag next for future alpha releases.

**Reply by mcollina** [reply]
released
